src/examples/java/com/devcycle/examples/OpenFeatureExample.java

```diff
@@ -0,0 +1,85 @@
+package com.devcycle.examples;
+
+import com.devcycle.sdk.server.local.api.DevCycleLocalClient;
+import com.devcycle.sdk.server.local.model.DevCycleLocalOptions;
+import dev.openfeature.sdk.*;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public class OpenFeatureExample {
+    public static void main(String[] args) throws InterruptedException {
+        String server_sdk_key = System.getenv("DEVCYCLE_SERVER_SDK_KEY");
+        if (server_sdk_key == null) {
+            System.err.println("Please set the DEVCYCLE_SERVER_SDK_KEY environment variable");
+            System.exit(1);
+        }
+
+        DevCycleLocalOptions options = DevCycleLocalOptions.builder().configPollingIntervalMS(60000)
+                .disableAutomaticEventLogging(false).disableCustomEventLogging(false).build();
+
+        // Initialize DevCycle Client
+        DevCycleLocalClient devCycleClient = new DevCycleLocalClient(server_sdk_key, options);
+
+        for (int i = 0; i < 10; i++) {
+            if (devCycleClient.isInitialized()) {
+                break;
+            }
+            Thread.sleep(500);
+        }
```

Comment:
It depends on how DevCycle users already use the product, but if you want to avoid something like this in OpenFeature, you could implement the [getState method](https://javadoc.io/doc/dev.openfeature/sdk/latest/index.html), and return `NOT_READY`, then do something like this in the `initialize` method of the provider, and setting READY and returning when you have initialized.